cmd/stack_select.go

```diff
@@ -62,12 +65,20 @@ func newStackSelectCmd() *cobra.Command {
 					return stackErr
 				}
 
-				stack, stackErr := b.GetStack(commandContext(), stackRef)
+				s, stackErr := b.GetStack(commandContext(), stackRef)
 				if stackErr != nil {
 					return stackErr
-				} else if stack != nil {
+				} else if s != nil {
 					return state.SetCurrentStack(stackRef.String())
 				}
+				//if create flag was passed and stack was not found, create it and select it
```

Comment:
Comments should be complete sentences. Starting with a capital letter and ending with a period. (And also having a space after the initial `//`.)